references/recognition/README.md

```diff
@@ -54,6 +53,7 @@ The order of entries in the json does not matter.
     ...
 }
 ```
+# labels.json
```

Comment:
I meant right after the line of the shell 🙃  like at the top within the code formatted section, above the opening bracket